project/AppDependencies.scala

```diff
@@ -43,7 +43,7 @@ object MainDependencies {
     "uk.gov.hmrc"         %% "play-reactivemongo"             % reactiveMongoVersion,
```

Comment:
this should be simple reactive mongo, play is too old and it should be a ~quick fix, 
(then change the val name)